### ISSUE TITLE
Icons don't work on localhost

### ISSUE THREAD

**Issue by alemacielsilva** [soflyy/breakdance-bugs#237]
Icons don't work on localhost. (using Beta 8)

![image](https://user-images.githubusercontent.com/21241956/184914134-3f18b7a1-bb2c-4a85-b629-dcd5da6b576c.png)

**Comment by louisreingold** [comment]
Something deleted the icons out of your wp_options table. Did you try to upload a very large amount of icons? Did you pull the site down from WP Engine or another host that runs aggressive database optimizations (for example, WP Engine auto-deletes anything in the wp_options table over 4Mb).

You can get the default icons back by going to Breakdance > Settings > Tools > Soft Reset in the WP admin.

**Comment by StaggerLeee** [comment]
Does not work for me and icons are there in wp_options table.
New install of Breakdance, no transfer. Localhost.

I have the same problem in 2 installations. Just did not notice before.

I checked third installation:

- Version 1.1-beta.2  =  Icons are there
- Update to Version 1.4.0-beta.2 = Icons are there
- Copied two icons tables from OK installation and imported in problematic one, OK now.

But, for some strange reason Version 1.4.0-beta.2 update went wrong in one installation. Plugin is trying to make files as executable in Linux and update goes wrong. Maybe that is the reason icons are missing. I set this particular breakdance folder with chmod to prevent making files as executable but it still insist on it. 
On other installation plugin does not make that what is strange. The same folders and subfolders for Apache.
